qa/integration-tests/src/test/java/io/camunda/it/utils/BrokerITInvocationProvider.java

```diff
@@ -45,71 +45,84 @@
  * The tests must take this into account, as the state in Zeebe and the exporter backend is not
  * reset between test cases.
  */
-public class BrokerWithCamundaExporterITInvocationProvider
+public class BrokerITInvocationProvider
```

Comment:
:thought_balloon: I guess we should find a better name at some point :smile:

Reply:
Yes! Ideas? :D